src/styles/_vars.scss

```diff
@@ -0,0 +1 @@
+$transition-time: 0.3s;
```

Comment:
I don't think you need to create a new file just for one variable. 
but if it works for you, i guess no problem